test/integration/bitbar/inspec/bitbar_devicepool_spec.rb

```diff
@@ -8,13 +8,6 @@
     end
   end
 
-  describe user('dhouse') do
```

Comment:
I think we should replace with you vs delete. I guess slightly more involved than just changing the login here, so let's punt to another PR.

Reply:
Will make a new PR.

Reply:
Made https://github.com/mozilla-platform-ops/ronin_puppet/pull/482